docs/tutorials/gettingstartedexample/gse-tap-basic.md

```diff
@@ -0,0 +1,231 @@
+| PXIe-1486 (4 In 4 Out - 953/954 SerDes)     | [LI-IMX490-FPDLINKIII](https://www.leopardimaging.com/product-category/autonomous-camera/ti-fpdlinkiii-cameras/li-imx490-fpdlinkiii/) |
+| PXIe-1487 (8 I - 9296A Deserializer)        | [LI-IMX490-GMSL2](https://www.leopardimaging.com/product-category/autonomous-camera/maxim-gmsl2-cameras/li-imx490-gmsl2/)             |
+| PXIe-1487 (4 In 4 Out - 9295A/9296A SerDes) | [LI-IMX490-GMSL2](https://www.leopardimaging.com/product-category/autonomous-camera/maxim-gmsl2-cameras/li-imx490-gmsl2/)             |
+
+> Note: The tap and acq modules used in this tutorial must have matching model numbers (i.e. 1486).
+
+## Initial Hardware Setup
+
+1.  First complete installation of hardware as described in the Getting Started Guide linked above.
+2.  Connect the camera to serial input channel 0 (SI0) on the tap module with a FAKRA cable.
+3.  Connect serial output channel (SO0) on the tap module to serial input channel (SI0) on the acq module with a FAKRA cable.
+4.  Connect the SI0 pin to the SO0 pin on the AUX POWER connector on the front panel of the tap module.
+
+## Initial Software Setup
+
+1.  First complete installation of software specified in the Getting Started Guide linked above.
+2.  Use the NI Example Finder to create a default project for your specific tap module.
+    - Open LabVIEW and click **Help -> Find Examples...**
+    
+    ![Open NI Example Finder](../../images/LabVIEW-Help-FindExamples.png)
+    
+    - In the NI Example Finder dialog, double-click **Hardware Input and Output -> FlexRIO -> Integrated IO -> Getting Started -> Getting Started FlexRIO Integrated IO.vi**
+    
+    ![NI Example Finder](../../images/NIExampleFinder-GSEFlexRIOWithIntegratedIO.png)
+
+    - In the FlexRIO with Integrated IO Project Creator dialog set the **Project Name** to <font face = "courier new">PXIe-148X Tutorial</font> and make the **FlexRIO Integrated IO** selection for the desired PXIe-148X interface module then click **OK**.
+    
+    ![FlexRIO with Integrated IO Project Creator](../../images/FlexRIOWithIntegratedIOProjectCreator-PXIe-148XTutorial.png)
+
+3.  Double click the Tap Example VI in the LabVIEW project.
+
+    ![Open Tap Example VI](../../images/PXIe-148X-TapGSE-Project.png)
+
+    > The opened front panel of the Getting Started Example is similar to the figure below.
+
+    ![Tap GSE Front Panel](../../images/PXIe-148X-TapGSE-FrontPanel.png)
+
+4. Open an Acquisition Example VI (see [Basic Acquisition Tutorials - Initial Software Setup](./gse-acq-basic.md#initial-software-setup)).
+    > Note: If the acq module is the same as the tap module, there is no need to create a new project. If the acq module is not the same as the tap module, a new project needs to be created for the acq module in order for the acq module FPGA bitfiles to be included in the project.
```

Comment:
Consider changing to "If the acq module is the same module type as the tap module," to remove ambiguity on whether someone can literally use the same physical module for acq and tap roles in the example (which we know they cannot)

Reply:
I made the change as suggested.